src/db.ts

```diff
@@ -432,98 +390,56 @@ export async function fetchAllMemeLabNFTs(orderBy?: string) {
   const tdhBlock = await fetchLatestTDHBlockNumber();
-  const sql = `SELECT ${ENS_TABLE}.display as ens, ${WALLETS_TDH_TABLE}.* FROM ${WALLETS_TDH_TABLE} LEFT JOIN ${ENS_TABLE} ON ${WALLETS_TDH_TABLE}.wallet=${ENS_TABLE}.wallet WHERE block=${tdhBlock};`;
-  const results = await execSQL(sql);
+  const sql = `SELECT ${ENS_TABLE}.display as ens, ${WALLETS_TDH_TABLE}.* FROM ${WALLETS_TDH_TABLE} LEFT JOIN ${ENS_TABLE} ON ${WALLETS_TDH_TABLE}.wallet=${ENS_TABLE}.wallet WHERE block=:block;`;
+  const results = await sqlExecutor.execute(sql, { block: tdhBlock });
   results.map((r: any) => (r.memes = JSON.parse(r.memes)));
   results.map((r: any) => (r.gradients = JSON.parse(r.gradients)));
   return results;
 }
 
 export async function fetchAllConsolidatedTDH() {
   const sql = `SELECT * FROM ${CONSOLIDATED_WALLETS_TDH_TABLE};`;
-  const results = await execSQL(sql);
+  const results = await sqlExecutor.execute(sql);
   results.map((r: any) => (r.memes = JSON.parse(r.memes)));
   results.map((r: any) => (r.gradients = JSON.parse(r.gradients)));
   return results;
 }
 
-export async function fetchConsolidatedTDHForWallet(w: any) {
-  let table;
-  let condition;
-  if (w.wallets) {
-    table = CONSOLIDATED_WALLETS_TDH_TABLE;
-    condition = `${table}.wallets LIKE '%${w.wallets[0]}%'`;
-  } else if (w.wallet) {
-    table = WALLETS_TDH_TABLE;
-    condition = `${table}.wallet=${mysql.escape(w.wallet)}`;
-  }
-
-  if (table) {
-    const sql = `SELECT tdh, boosted_tdh, tdh__raw, boost, memes, gradients FROM ${table} WHERE ${condition} ORDER BY BLOCK DESC LIMIT 1`;
-    const results = await execSQL(sql);
-    if (results.length > 0) {
-      const consolidated = results[0];
-      consolidated.memes = consolidated.memes
-        ? JSON.parse(consolidated.memes)
-        : [];
-      consolidated.gradients = consolidated.gradients
-        ? JSON.parse(consolidated.gradients)
-        : [];
-      return consolidated;
-    }
-  }
-  return null;
-}
-
-export async function fetchTDHHistoryForWallet(wallet: string, block: number) {
-  const sql = `SELECT * FROM ${TDH_HISTORY_TABLE} WHERE LOWER(${TDH_HISTORY_TABLE}.wallets) LIKE '%${wallet.toLowerCase()}%' AND block=${block}`;
-  const results = await execSQL(sql);
-  if (results.length > 0) {
-    const consolidated = results[0];
-    consolidated.memes = consolidated.memes
-      ? JSON.parse(consolidated.memes)
-      : [];
-    consolidated.gradients = consolidated.gradients
-      ? JSON.parse(consolidated.gradients)
-      : [];
-    return consolidated;
-  }
-  return null;
-}
-
 export async function fetchConsolidationDisplay(
   myWallets: string[]
 ): Promise<string> {
-  const sql = `SELECT * FROM ${ENS_TABLE} WHERE wallet in (${mysql.escape(
-    myWallets
-  )})`;
-  const results = await execSQL(sql);
+  const sql = `SELECT * FROM ${ENS_TABLE} WHERE wallet IN (:wallets)`;
+  const results = await sqlExecutor.execute(sql, {
+    wallets: myWallets.join(',')
```

Comment:
I think you dont have to do this join thing. sqlExecutor takes care of this when you pass just an array. Look at db.ts:201

Reply:
you're right